src/components/List/__stories__/index.stories.tsx

```diff
@@ -518,3 +521,80 @@ PaginationLoadingPageCount.decorators = [
     )
   },
 ]
+
+export const Animated = Template.bind({})
```

Comment:
Excellent demo ;)
An UX review should be as important as the code review for this!

Reply:
@adriengibrat Probably animations will be more complete once design system process will start and we will have more from design team :)